

You are applying for a job as a data scientist at a tech company. Please carefully read the text in triple quotes and complete the tasks.

'''

Your Resume:
Name: Sijia Cui

Position: Bioinformatics Project Scientist Affiliation: Cedars Sinai Medical Center
Areas of Expertise:

    Independent Research

Highly skilled in utilizing public data for data mining, independently designing algorithms, and integrating deep learning and machine learning techniques to address biological problems.

Project Example: Utilized publicly available single-cell data on Inflammatory Bowel Disease (IBD) and applied a modified Variational Autoencoder (VAE) model for label transfer. Combined with currently published bulk RNA-seq datasets using Graphic neutral network punishment, this approach aided in identifying key cell subpopulations and important genes associated with various IBD phenotypes.

    Spatial single cell analysis: Proficient in analyzing spatial single-cell data, including CODEX and 10x genomics platforms, to perform cellular segmentation and identify cellular neighborhoods and interactions.

    Multi-omics single-cell analysis: Skilled in integrating diverse omics data, such as scRNA-seq, scATAC-seq, and scTCR-seq, to provide a comprehensive understanding of cellular heterogeneity and function.

Project Example: Please see the cancer cell paper under review "Multi-omics analysis dissecting the single-cell architecture of late-relapsed hepatocellular carcinoma"

    Multi-omics bulk analysis: Experienced in integrating diverse omics data, such as genomics, transcriptomics, and epigenomics, to provide a comprehensive understanding of cancer intra- or inter- heterogeneity.

Prorject Example: Please see the nature communication paper "Tracking the evolution of esophageal squamous cell carcinoma under dynamic immune selection by multi-omics sequencing"
Technical Skills:

Programming Languages: Solid proficiency in Python and R, with practical experience in Pytorch and Tensorflow.

Data Science and Analysis: Highly skilled in deep learning, machine learning, and statistical analysis. Familiar with advanced techniques such as Variational Autoencoders, Generative Adversarial Networks, Transfer Learning, and Transformers, demonstrating a strong ability to develop and apply complex models to biological data.
Accomplishments

    Cui S*, Nicholas, Tracking the evolution of esophageal squamous cell carcinoma under dynamic immune selection by multi-omics sequencing. Nature communication, 2023 (As first author)
    Duan M*, Hao J*, Cui S*, et al. Diverse modes of clonal evolution in HBV-related hepatocellular carcinoma revealed by single-cell genome sequencing.[J]. Cell Research, 2018 (As first author)
    Zhang, K., Cui, S., et al. i-GSEA4GWAS: a web server for identification of pathways/gene sets associated with traits by applying an improved gene set enrichment analysis to genome-wide (As Co-first author)
    Zhang, K., Chang, S., Cui, S., et al. ICSNPathway: identify candidate causal SNPs and pathways from genome-wide association study by one analytical framework. Nucleic Acids Res, 2011(Web Server issue) [IF: 16]
    Zhu, X., Chang, S., Fang, K., Cui, S., et al. (2009). MyBASE: a database for genome polymorphism and gene function studies of Mycobacterium. BMC Microbiol [IF: 2.729, 5-year IF: 3.251)

Under Review:

    Zhang Q, Cui S, et al. Multi-omics analysis dissecting the single-cell architecture of late-relapsed hepatocellular carcinoma. Cancer Cell, 2024 (As Co-first author)
    Integrative Analysis of Single-Cell and Bulk RNA Sequencing to Identify Cell Types Associated with HCC phenotypes. Nature Communication, 2024 (As Co-first author)
    Single-cell spatial immune landscapes of primary HCC tumours. (In Submission, As Co-first author)

Work History
Cedars Sinai Medical Center

Bioinformatics Project Scientist Duration: January 2023 - Present Location: Los Angeles, California

Led a pioneering study in spatial single-cell analysis for Intestinal Bowel Disease (IBD), significantly advancing our understanding of its underlying mechanisms.

Developed an innovative Transformer-based model to analyze CyToF blood sample data, markedly improving diagnostic accuracy for IBD patient cohorts.

Executed an extensive single-cell RNA sequencing (scRNA-seq) project for the IBD Atlas study, yielding groundbreaking insights into the disease.

Employed Graph Convolutional Networks (GCN) for the sophisticated identification of cellular neighborhoods based on the CosMx NanoString dataset.

Applied Variational Autoencoders (VAE) for effective data augmentation, predicting potential cellular neighborhoods using the scRNA-seq dataset, demonstrating an innovative approach to CosMx data analysis.
Baylor College of Medicine

Postdoctoral Fellow Duration: April 2022 - January 2023 Location: Houston, Texas

Led the scRNA-seq analysis for an Esophageal Cancer study, advancing cancer research. Orchestrated a multi-omics analysis in Esophageal Cancer research, integrating diverse data sets for a holistic understanding of the disease.
Beijing Chineo Medical Technology Co., Ltd.

Senior Bioinformatics Scientist Duration: July 2016 - April 2022 Location: Beijing, China

Oversaw CAR-T (T-cell therapies and cellular therapies) projects for blood and solid tumor drug development, focusing on neoantigen recognition using bulk and single-cell sequencing. Developed a high-accuracy Convolutional Neural Network (CNN) model for liver cancer patient diagnosis using the CyToF dataset (AUC > 0.9). Pioneered a multi-omics single-cell sequencing pipeline (including scRNA-seq, single-cell ATAC-seq, and TCR-seq) to elucidate immune activation mechanisms and aid novel drug development.
Novogene Biotechnology Co., Ltd.

Bioinformatics Engineer Duration: September 2014 - July 2016 Location: Beijing, China

Established an automated analysis pipeline for next-generation sequencing in cancer research, enhancing the understanding of tumor evolution and disease progression. Innovated a rapid and precise virus integration site (VIS) detection method, VirusFaster, capable of achieving single base resolution in genome sequencing data analysis.
Education

University of Chinese Academy of Sciences, Beijing Ph.D. in Behavioral Genetics Duration: September 2009 - July 2014

Focused on predictive modeling using advanced computational techniques, including Random Forest and Support Vector Machine, to identify high-risk loci from Genome-Wide Association Studies (GWAS) data. Developed a comprehensive bioinformatics database and web server, showcasing proficiency in Java and MySQL.

Job Description:

'''





 Overview

Fred Hutchinson Cancer Center is an independent, nonprofit organization providing adult cancer treatment and groundbreaking research focused on cancer and infectious diseases. Based in Seattle, Fred Hutch is the only National Cancer Institute-designated cancer center in Washington.


With a track record of global leadership in bone marrow transplantation, HIV/AIDS prevention, immunotherapy and COVID-19 vaccines, Fred Hutch has earned a reputation as one of the world’s leading cancer, infectious disease and biomedical research centers. Fred Hutch operates eight clinical care sites that provide medical oncology, infusion, radiation, proton therapy and related services, and network affiliations with hospitals in five states. Together, our fully integrated research and clinical care teams seek to discover new cures to the world’s deadliest diseases and make life beyond cancer a reality.


At Fred Hutch we value collaboration, compassion, determination, excellence, innovation, integrity and respect. These values are grounded in and expressed through the principles of diversity, equity and inclusion. Our mission is directly tied to the humanity, dignity and inherent value of each employee, patient, community member and supporter. Our commitment to learning across our differences and similarities make us stronger. We seek employees who bring different and innovative ways of seeing the world and solving problems. Fred Hutch is in pursuit of becoming an anti-racist organization. We are committed to ensuring that all candidates hired share our commitment to diversity, anti-racism and inclusion.

 

The Senior Bioinformatician will lead data analysis projects supporting genomic, transcriptomic, clinical and spatial applications, including creating and managing a data structure/warehouse for the Fong Lab and the Immunotherapy Integrated Research Center (IIRC). The position requires expertise with advanced workflows including spatial biology; single-cell analysis; epigenetic profiling; precision medicine; microbial and metagenomic analysis; and novel applications of third generation sequencing. The Senior Bioinformatician partners with faculty, post-doctoral fellows, and research staff to understand specific technical requirements, effectively communicate complex bioinformatics concepts to non-specialists, and deliver timely and tailored solutions. In addition, this position requires continual assessment of emerging techniques and analysis tools to leverage the most appropriate and up-to-date methods. The Senior Bioinformatician actively incorporates new techniques into workflows and mentors staff to expand the team’s capabilities.
Responsibilities

Responsibilities span experimental design, data analysis, integration of high-dimensional data from multiple assay types and providing text and figures to document results. The Senior Bioinformatician provides a consultative and collaborative environment for each stakeholder, with personalized support and suggestions tailored to the needs of each project. Acting as subject matter expert and mentoring other team members is vital to this position. The Senior Bioinformatician acts independently to maintain these relationships, prioritizes multiple simultaneous projects, possesses exceptional communication and organizational abilities, and supports stakeholders with updates and documentation.

    Lead all phases of data analysis, integration, and reporting for multiple simultaneous investigator-initiated projects.
    Review emerging assays and computational techniques to keep the team current and to extend our capabilities into new areas.
    Delegate data analysis tasks to and provide mentoring for other members of the Bioinformatics team as appropriate.
    Integrate and harmonize diverse high-dimensional data sets (for example, multi-omic single-cell with spatial gene expression).
    Collaborate with researchers developing novel assays and analysis methods (for example, artificial intelligence, machine learning).
    Communicate and present results to stakeholders.
    Provide figures and written sections to document methods and results for manuscripts, presentations, and grant applications.
    Representing the team at poster sessions or other internal and external forums.
    Collaborate with colleagues in the Genomics Shared Resource and the Hutch Data Core to ensure the integrity and consistency of primary data, track experimental methods and metadata, define standardized analysis pipelines, and provide training and support as stakeholders analyze and interpret results.
    Train laboratory members and collaborators on analytic workflows.
    Upload and maintain datasets, workflows, and code (for example GEO, Github).

Qualifications

MINIMUM QUALIFICATIONS:

    MS or PhD in bioinformatics, computational biology, genetics, or related field, or equivalent combination of degree and experience.
    Minimum of 8 years professional experience including direct experience in analysis of large, complex sequence-based molecular data sets.
    Direct experience must include several of these assay types: whole exome sequencing, targeted amplicon sequencing, best-practice germline and somatic variant calling from exome capture data, bulk RNA-seq data with multiple contrasts, analysis of multimodal single-cell expression and immune repertoire profiling, gene set enrichment, and integration of data across multiple modalities (e.g., TCR-seq, ATAC-seq, and CRISPR/Cas9 knockout screens).
    Demonstrated expertise in integrative multi-omics analysis of these complex data types.
    Strong record of contribution to publications in peer-reviewed journals.
    Solid grounding in molecular biology, genetics, and statistical methods.
    Demonstrated fluency in R or Python 3 are essential.
    Proficiency with Linux/Unix shell scripting (e.g., bash).
    Facility with commonly used Bioconductor packages, ggplot, tidyverse etc.
    Ability to generate and customize common data visualizations (PCA plots, volcano plots, Circos plots, etc).
    Familiarity with workflow and scheduling software (e.g., Slurm).
    Excellent written and verbal communication skills.
    Exceptional organizational skills, including the ability to independently drive and prioritize multiple competing engagements with researchers that may span several months.

    Write a cover letter that explains why you are a good fit for the position.